storage/dkg_keys.go

```diff
@@ -5,6 +5,16 @@ import (
 )
 
 type DKGKeys interface {
+	// insert the DKG private key to database when DKG was completed, and a DKG private key was successfully generated.
 	InsertMyDKGPrivateInfo(epochCounter uint64, key *dkg.DKGParticipantPriv) error
-	RetrieveMyDKGPrivateInfo(epochCounter uint64) (*dkg.DKGParticipantPriv, error)
+
+	// insert a record in database to indicate that when DKG was completed, the node failed DKG, there is no DKG key generated.
+	InsertNoDKGPrivateInfo(epochCounter uint64) error
```

Comment:
😢 missing possible error returns during normal operation here as well as in the interface.

Reply:
👍 Just a quick comment like: "During normal operations, no error returns are expected." would clarify